tox.ini

```diff
@@ -85,6 +85,7 @@ commands_pre =
     # strictly only need to uninstall pytest-qt (which will raise without a backend)
     # the rest is for good measure
     headless: pip uninstall -y pytest-qt qtpy pyqt5 pyside2
+    !headless: pip install pydantic<1.8
```

Comment:
This can probably be removed with a rebase?

Reply:
done.